.github/pull_request_template.md

```diff
@@ -4,10 +4,10 @@ Please include a summary of the change. Please also include relevant motivation
 
 # Checklist:
 
-- [ ] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
+- [ ] My code follows the [style guidelines](https://github.com/dimaqq/sdcore-amf-k8s-operator/blob/main/CONTRIBUTING.md) of this project
```

Comment:
We probably don't want to point to your fork of the project.

Reply:
ugh, sorry!

Reply:
fixed!